samtranslator/translator/translator.py

```diff
@@ -51,7 +51,7 @@ def _get_function_names(self, resource_dict, intrinsics_resolver):
                     # adds to the function_names dict with key as the api_name and value as the function_name
                     if item.get("Type") == "Api" and item.get("Properties") and item.get("Properties").get("RestApiId"):
                         rest_api = item.get("Properties").get("RestApiId")
-                        if type(rest_api) == dict or isinstance(rest_api, dict):
+                        if isinstance(rest_api, dict):
```

Comment:
Resolved conflicts. I also updated this section- it turns out that an `OrderedDict` is an instance of `dict`, so this works for both cases we wanted to cover.

Reply:
Great, appreciate it!